### ISSUE TITLE
Prevent a local muted user from sending traffic

### ISSUE THREAD

**Issue by promi** [mumble-voip/mumble#3973]
When a user is locally muted audio packets are still received from that user. That is not optimal for users with metered internet connections as it causes unnecessary traffic.

It would be nice if a client could subscribe and unsubscribe from audio being send by a specific other user.

PS: #2119 is a somewhat related issue

**Comment by Krzmbrzl** [comment]
So the scenario would be that you are in a channel with User A and B. Now you mute A. You still want to hear B though.

And in this case whenever A speaks, you'll get the audio packets sent to you although you'll never hear them (as you muted A).
Is that correct?

**Comment by promi** [comment]
Yes, that is correct. My client is (locally) muting user A. When A speaks I still get his audio packets.
